src/MusicStore.Standalone/Program.cs

```diff
@@ -44,9 +44,10 @@ public static void Main(string[] args)
                 builder.UseKestrel();
             }
 
-            var host = builder.Build();
-
-            host.Run();
+            using (var host = builder.Build())
```

Comment:
This doesn't seem to be the pattern that's used in all the project templates. The templates all just call `.Build()` and then `.Run()` without disposal:
* https://github.com/aspnet/Templates/blob/dev/src/BaseTemplates/WebAPI/Program.cs#L20-L22
* https://github.com/aspnet/Templates/blob/dev/src/BaseTemplates/EmptyWeb/Program.cs#L19-L21
* https://github.com/aspnet/Templates/blob/dev/src/BaseTemplates/StarterWeb/Program.cs#L19-L21

I would prefer keeping this to match what the templates do as I don't think there's any need to dispose anything here (the app will exit and clean up all resources after the last line anyway).